devel/git-sizer/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            github git-sizer 1.0.0 v
+categories              devel
+platforms               darwin
+supported_archs         noarch
+maintainers             @gerardsoleca openmaintainer
+license                 MIT
+
+description             Compute various size metrics for a Git repository, \
+                        flagging those that might cause problems
+long_description        ${description}
```

Comment:
Could the long description give more information than this? I don't really know what "flagging those that might cause problems" etc. means.